js_modules/dagit/packages/core/src/app/QueryRefresh.tsx

```diff
@@ -33,7 +33,11 @@ export interface QueryRefreshState {
  *   or the default cache-first fetchPolicy.
  *
  */
-export function useQueryRefreshAtInterval(queryResult: QueryResult<any, any>, intervalMs: number) {
+export function useQueryRefreshAtInterval(
+  queryResult: QueryResult<any, any>,
+  intervalMs: number,
+  enabled = true,
```

Comment:
This looks good to me, but I think we achieve this general effect elsewhere by setting intervalMs to a huge number -- we might want to switch those callsites to use this new param